src/main/scala/ch/openolitor/stammdaten/repositories/StammdatenRepositoryQueries.scala

```diff
@@ -1476,7 +1476,7 @@ trait StammdatenRepositoryQueries extends LazyLogging with StammdatenDBMappings
       select
         .from(korbMapping as korb)
         .innerJoin(lieferungMapping as lieferung).on(lieferung.id, korb.lieferungId)
-        .where.eq(korb.aboId, aboId).and.not.eq(korb.status, WirdGeliefert)
+        .where.eq(korb.aboId, aboId)
```

Comment:
If this is what we want, let's change the method name to getKoerbeQuery and the one in the read repository to getKoerbe.

Reply:
Done